subiquity/server/controllers/filesystem.py

```diff
@@ -351,7 +352,7 @@ async def v2_add_boot_partition_POST(self, disk_id: str) \
         self.add_boot_disk(disk)
         return await self.v2_GET()
 
-    async def v2_add_partition_POST(self, data: ModifyPartitionV2) \
+    async def v2_add_partition_POST(self, data: AddPartitionV2) \
```

Comment:
Er. Shouldn't this function actually use data.gap now?

Reply:
:facepalm: yes